homeassistant/components/media_extractor/__init__.py

```diff
@@ -106,7 +108,18 @@ def extract_and_send(self) -> None:
 
     def get_stream_selector(self) -> Callable[[str], str]:
         """Return format selector for the media URL."""
-        ydl = YoutubeDL({"quiet": True, "logger": _LOGGER})
+        cookies_file = os.path.join(
+            self.hass.config.config_dir, "media_extractor", "cookies.txt"
+        )
```

Comment:
I got the advice to use path lib for this, I am currently in the train so can't really check https://docs.python.org/3/library/pathlib.html

Reply:
I don't see any advantage in this but if that's the prefered way I'll use it. It means creating a complex type that has to be converted back to a string to be used.